src/Google/Ads/GoogleAds/Util/V1/GoogleAdsFailures.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Ads\GoogleAds\Util\V1;
+
+use Google\Protobuf\Any;
+use Google\Rpc\Status;
+use Google\Ads\GoogleAds\V1\Errors\GoogleAdsFailure;
+
+final class GoogleAdsFailures
+{
+    /**
+     * Unpacks a single GoogleAdsFailure from an Any instance.
+     *
+     * @param Any an Any instance to unpack
+     * @return GoogleAdsFailure
+     */
+    public static function fromAny(Any $any)
+    {
+        return $any->unpack();
```

Comment:
A thought here - do you want to check that the unpacked message is actually the expected GoogleAdsFailure class, and if not, to either return null or throw an exception? Right now you could call this method on an Any object containing anything, and it will succeed and give you back that object.

Reply:
Good point, especially since the clients are going to assume that the returned object is a GoogleAdsFailure. 

I think in this case throwing an exception would be the best solution, as unpacking a message that is not a GoogleAdsFailure here is really a wrong usage of this function. 

However, I don't have a strong opinion on what exception we should throw here - InvalidArgumentException? A custom exception? @fiboknacky, WDYT?

Reply:
InvalidArgumentException SGTM.

Reply:
Done.